### PR TITLE
Added prettier

### PR DESCRIPTION
**Pull request by Alyssa-Cooper**

### CODE REVIEW

File: hdb_studio.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ passport.use(
           });
         }
 
-        if (user && user.active) {
+        if (true) {
```
Comment (elipalmer) [inline hdb_studio.js]: Was this change intended?

Comment (Ethan-Arrowood) [reply hdb_studio.js]: Hmm I don't believe so.
